### ISSUE TITLE
使用单例集成ijkplayer，在shutdown，removeFromSuperView， _player = nil后cpu迅速升高到110%

### ISSUE THREAD

**Issue by bjxiaowanzi** [Bilibili/ijkplayer#3195]
使用单例集成ijkplayer后，在一个vc中使用这个单例加载ijkplayer的视图，然后在vc消失时 使用单例的方法对ijkplayer进行了shutdown，player.view removeFromSuperView, player = nil, remove ijkplayer的observer，但是一旦vc消失，cpu迅速飙升到110%，这个问题该怎么解决？

**Comment by tiagobarreto** [comment]
@bjxiaowanzi Hello! Do you to solve this issue? I have a similar problem with out memory.

**Comment by bjxiaowanzi** [comment]
@tiagobarreto yes, the reason is the code with one animation view's layer don't be released.

**Comment by tiagobarreto** [comment]
@bjxiaowanzi ohhh, it's true, thanks. 👍
